datasets/ladi.yaml

```diff
@@ -0,0 +1,56 @@
+Name: Low Altitude Disaster Imagery (LADI) Dataset
+Description: The Low Altitude Disaster Imagery (LADI) Dataset consists of human and machine annotated airborne images collected by the Civil Air Patrol in support of various disaster responses from 2015-2019. The initial release of LADI focuses on the Atlantic hurricane seasons and coastal states along the Atlantic Ocean and Gulf of Mexico. Annotations are included for major hurricanes of Harvey, Maria, and Florence. Two key distinctions are the low altitude, oblique perspective of the imagery and disaster-related features, which are rarely featured in computer vision benchmarks and datasets.
+Documentation: https://github.com/LADI-Dataset/ladi-overview
+Contact:  ladi-dataset-admin@mit.edu
+ManagedBy: "[MIT Beaver Works](https://beaverworks.ll.mit.edu/)"
+UpdateFrequency: Periodically
+License: Creative Commons Attribution 4.0 International (CC BY 4.0) 
+Tags:
+  - aws-pds
+  - aerial imagery
+  - coastal
+  - computer vision
+  - disaster response
+  - earth observation
+  - earthquakes
+  - geospatial
+  - imaging
+  - image processing
+  - infrastructure
+  - land
+  - machine learning
+  - mapping
+  - natural resource
+  - seismology
+  - transportation
+  - urban
+  - water
+Resources:
+  - Description: LADI dataset: images and labels
```

Comment:
We need to add quotes around "LADI dataset: images and labels" here since there is a : there that confuses the yaml parser

Reply:
Addressed in commit 33e04f0